FrontEndLib/BumpObstacleEffect.cpp

```diff
@@ -73,28 +73,51 @@ CBumpObstacleEffect::~CBumpObstacleEffect()
 		SDL_FreeSurface(this->pEraseSurface);
 }
 
+#include "Screen.h"
```

Comment:
Move #include to top of file.

Reply:
That was used for some debugging earlier, removed.